docs/application/dotnet/guides/multimedia/media-controller.md

```diff
@@ -23,10 +23,11 @@ The main media controller features include:
 
 To enable your application to use the media controller functionality:
 
-1.  To use the [Tizen.Multimedia.Remoting.MediaControlServer](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Multimedia.Remoting.MediaControlServer.html) class, the application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:
+1.  To use the [Tizen.Multimedia.Remoting.MediaController](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Multimedia.Remoting.MediaController.html) and [Tizen.Multimedia.Remoting.MediaControlServer](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Multimedia.Remoting.MediaControlServer.html) class, the application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:
```

Comment:
Question: In the privileges we have client and server why is Tizen.Multimedia.Remoting.MediaControlServer changed to Tizen.Multimedia.Remoting.MediaController any specific reason?